src/main/java/manga/Manga.java

```diff
@@ -38,6 +52,6 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(mangaName, author);
+        return Objects.hash(mangaName, author, deadline);
```

Comment:
I think as per the previous comment on the `equals` method, we can leave out the `deadline` during comparison.